src/ir_Fujitsu.cpp

```diff
@@ -184,7 +184,13 @@ void IRFujitsuAC::checkSum(void) {
       }
     }
     if (_model != fujitsu_ac_remote_model_t::ARRY4) {
-      if (_model != fujitsu_ac_remote_model_t::ARREW4E) _.Clean = false;
```

Comment:
Outer if could be removed and included in the case?

Reply:
I don't think so. I checked at the time I wrote it, and re-checked it now. I'm fairly sure that is the correct code because other models need the `_.Filter = false;` statement.
If you think the logic is the same, please explain because I could be missing something.